### PR TITLE
[Issue NexClipper#101] Change Toggle Button design and Rearrange it

### PR DESCRIPTION
**Pull request by Joylish**
Issue NexClipper#101

### Now
image

### After edited
darkmodeGIF

### To Think More
- [ ]  Apply user preset mode

### CODE REVIEW

File: src/components/Darkmode/ToggleButton.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { changeDarkTheme } from "../../store/actions/exporterActions";
+
+// import Toggle from "react-toggle";
```
Comment (BaeJi77) [inline src/components/Darkmode/ToggleButton.js:6]: I think you should remove unused import commanded.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import Routes from "./Routes";
+
+import ToggleButton from "./components/Darkmode/ToggleButton";
```
Comment (BaeJi77) [inline src/index.js:5]: I think you should name `ToggleButton` to more specific name. like `DarkModeToggleButton`.

**Comment by ralfyang** [comment]
Very nice! thank you! :)

**Comment by ralfyang** [comment]
I just checked in dev env. it'll be merge directly.

File: src/components/Darkmode/ToggleButton.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { changeDarkTheme } from "../../store/actions/exporterActions";
```
[the remark this answers is not in the document]
Comment (Joylish) [reply src/components/Darkmode/ToggleButton.js:4]: Thanks to comment !!
I will solve it by webpack and will include it in the next PR 🐾

File: src/components/Darkmode/ToggleButton.js (hunk 1 above)
Comment (Joylish) [reply src/components/Darkmode/ToggleButton.js:6]: You are sharp! Thanks for reading my code.
I will remove this code as well and include it in the next pr.

File: src/index.js (hunk 2 above)
Comment (Joylish) [reply src/index.js:5]: Oh, that's a more intuitive name! Naming is difficult... 
This pr has already been merged, so I'll merge all the parts you mentioned and post it in the next pr. 
I wish you comment on the next pr 👻🌠

**Pull request merged**